client/components/domains/registrant-extra-info/test/fr-form.js

```diff
@@ -0,0 +1,83 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import { expect } from 'chai';
+
+/**
+ * Internal dependencies
+ */
+import { sanitizeVat } from '../fr-form';
+
+describe( 'fr-form', () => {
+	describe( 'VAT Sanitation', () => {
+		test( 'should clean invalid characters', () => {
+			const vatPatterns = [
+				{ before: 'FR 99999999999', after: 'FR99999999999' },
+				{ before: ' FR99999999999', after: 'FR99999999999' },
+				{ before: 'FR99999999999 ', after: 'FR99999999999' },
+				{ before: ' FR99 999 9 99999 ', after: 'FR99999999999' },
+				{ before: ' FR99@99!99&999()99 ', after: 'FR99999999999' },
+			];
+
+			vatPatterns.forEach( ( { before, after } ) =>
+				expect( sanitizeVat( before ) ).to.eql( after )
+			);
+		} );
+
+		test( 'should preserve valid patterns', () => {
+			// [ pattern, optionalDescription ]
+			// These are just patterns, not valid numbers.
+			const validVatPatterns = [
```

Comment:
We already have this list in `components/domains/registrant-extra-info/test/fr-validate-contact-details.js`, I don't like the fact that it's duplicated here - maybe we could refactor it in a future iteration so that they are shared between these two files?